app/models/json_web_token.rb

```diff
@@ -0,0 +1,9 @@
+class JsonWebToken
```

Comment:
models does not seem to be the right place for this.
you can place this in lib.

Reply:
i'll review the rest tomorrow.